example/package.json

```diff
@@ -3,25 +3,26 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "clear": "node ./node_modules/react-native/local-cli/cli.js start --reset-cache",
+    "clear": "react-native start --reset-cache",
     "install-pods": "cd ios && pod install && cd ..",
-    "start": "node ./node_modules/react-native/local-cli/cli.js start",
+    "start": "react-native start",
     "test": "jest",
     "flow": "flow"
   },
   "dependencies": {
-    "react": "16.8.6",
-    "react-native": "0.60.3"
+    "react": "16.9.0",
+    "react-native": "0.60.5",
+    "react-native-share": "^2.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.5.4",
     "@babel/runtime": "^7.5.4",
     "babel-jest": "^24.8.0",
-    "flow-bin": "0.98.1",
+    "flow-bin": "0.105.2",
     "jest": "^24.8.0",
```

Comment:
not in the template. would be great though right? I worked so hard to get the jest-haste-map patches pushed out so node.js v12 wouldn't take 2 minutes to start up

Reply:
Wow, you're right, upgrade helper does not show it.
However, if you generate a new RN project (0.60.5) using `npx react-native init` today, **it will contain** the newer jest and babel-jest (`^24.9.0`) dependencies. Super confusing.

Reply:
even better yarn vs npm behave differently in this situation. For yarn if you just delete yarn.lock and then run yarn, you'll get jest-everything at 24.9.0 (and thus not have the jest-haste-map-tickles-a-node-v12.4-through-v12.8.1 bug), but with npm install it doesn't work. Much pain